packages/commonwealth/client/scripts/views/pages/Community/Members/GroupsSection/GroupsSection.tsx

```diff
@@ -73,9 +72,10 @@ const GroupsSection = ({
                 requirementCondition: 'More than', // hardcoded in api
               }))}
               requirementsToFulfill="ALL" // api doesn't return this
-              isJoined={(group.members || []).find(
-                (x) => x?.address?.address === app.user.activeAccount.address
-              )}
+              isJoined={(group.members || []).find((x) => {
+                if (!app.user.activeAccount) return;
```

Comment:
nice spot here, i think we should also check for null `app?.user?.activeAccount` -- it breaks if not login I think